src/servers/src/postgres/handler.rs

```diff
@@ -201,16 +179,39 @@ fn type_translate(origin: &ConcreteDataType) -> Result<Type> {
 
 #[async_trait]
 impl ExtendedQueryHandler for PostgresServerHandler {
+    type Statement = String;
+    type QueryParser = NoopQueryParser;
+    type PortalStore = MemPortalStore<Self::Statement>;
+
+    fn portal_store(&self) -> Arc<Self::PortalStore> {
+        self.portal_store.clone()
+    }
+
+    fn query_parser(&self) -> Arc<Self::QueryParser> {
+        self.query_parser.clone()
+    }
+
     async fn do_query<C>(
         &self,
         _client: &mut C,
-        _portal: &Portal,
+        _portal: &Portal<Self::Statement>,
         _max_rows: usize,
     ) -> PgWireResult<Response>
     where
         C: ClientInfo + Unpin + Send + Sync,
     {
-        unimplemented!()
+        unimplemented!("Extended Query is not implemented on this server.")
```

Comment:
Better return some error instead of panic, because this `unimplemented` could be triggered by user input. Same to line 214.